src/equality/eqchk_normalizer.ml

```diff
@@ -226,11 +216,11 @@ and normalize_heads_type ~strong sgn nrm ty0 =
      let heads = get_type_heads nrm (Ident s) in
      let args_eq_args', Normal args' = normalize_arguments ~strong sgn nrm heads args in
      let ty1 =
-       let jdg1 = deopt (rap_fully_apply (Nucleus.form_constructor_rap sgn s) args') in
-       deopt (Nucleus.as_is_type jdg1) in
+       let jdg1 = deopt (rap_fully_apply (Nucleus.form_constructor_rap sgn s) args') "cannot apply arguments to type constructor" in
```

Comment:
Let's try to keep the length of lines below 100 (ideally 80) characters when possible.